hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopFsGenerator.java

```diff
@@ -63,6 +63,12 @@ public class HadoopFsGenerator extends BaseFreonGenerator
       defaultValue = "4096")
   private int copyBufferSize;
 
+  @Option(names = {"--sync"},
+      description = "Type of operation to execute after a write. Supported " +
+      "options include NONE (default), HFLUSH and HSYNC",
+      defaultValue = "none")
```

Comment:
```suggestion
      defaultValue = "NONE")
```

`none` gives error if the option is omitted:

```
$ ozone freon dfsg --path ofs://om/vol1/buck1/dir1 -s 1024 -t1 -n1
...
Invalid value for option '--sync': expected one of [NONE, HFLUSH, HSYNC] (case-sensitive) but was 'none'
```